types/mergician.d.ts

```diff
@@ -0,0 +1,163 @@
+declare module 'mergician' {
+
+	type ObjectLiteral = Record<any, any>
+
+	export interface MergicianOptions {
+
+		// ----
+		// Keys
+		// ----
+
+		/**
+		 * Exclusive array of keys to be merged (others are skipped).
+		 */
+		onlyKeys?: string[];
+
+		/**
+		 * Array of keys to skip (others are merged).
+		 */
+		skipKeys?: string[];
+
+		/**
+		 * Merge only keys found in multiple objects (ignore single occurrence keys).
+		 */
+		onlyCommonKeys?: boolean;
+
+		/**
+		 * Merge only keys found in all objects
+		 */
+		onlyUniversalKeys?: boolean;
+
+		/**
+		 * Skip keys found in multiple objects (merge only single
+		 *     occurrence keys)
+		 */
+		skipCommonKeys?: boolean;
+
+		/**
+		 * Skip keys found in all objects (merge only common keys)
+		 */
+		skipUniversalKeys?: boolean;
+
+		// ------
+		// Arrays
+		// ------
+
+		/**
+		 * Merge array values at the end of existing arrays
+		 */
+		appendArrays?: boolean;
```

Comment:
If option names and types are defined by the `MergicianOptions` interface, I would expect the following to throw an error:

```js
const customMergeFn = mergician({
  appendArrays: 1
});
```

Unfortunately, no error is displayed in VS Code or when during compilation.